### ISSUE TITLE
MVC Html.Action() and Html.RenderAction()

### ISSUE THREAD

**Issue by Scippy** [INNO-Software/DnnJobBoard#3]
Hi, I'm looking at your project for better understand how MVC work with DNN.
About the Url.Action to show "View Details" I can see that the page is full reloaded, I'd like to know if there is any way to render a view with controllor injecting the code in the page like Html.Action() or Html.RenderAction()
Thanks!

**Comment by INNO-Software** [comment]
Yes, this project was made when I was establishing an approach to MVC in DNN. Now I'd use JavaScript that calls into the services layer inside DNN. So this would prevent a postback action. That said, it's not calling Html.Action() or Html.RenderAction() specifically.
